src/coprocessor/dag/rpn_expr/impl_arithmetic.rs

```diff
@@ -220,4 +321,152 @@ mod tests {
             assert_eq!(output, expected, "{:?}, {:?}", output, expected);
         }
     }
+
+    #[test]
+    fn test_mod_int() {
+        let tests = vec![
+            (ModInt, Some(13), Some(11), Some(2)),
```

Comment:
If all test cases in a set have the same func sig, we may omit it and write it in the parameter of evaluate.
I will change my test code too.

Reply:
OK, I will change all the tests in the file.

Reply:
@sticnarf  FYI I did this for you in https://github.com/tikv/tikv/pull/4729/files#diff-633e80b547d74744a71fe178a057e8fe when I was fixing the conflict XD No need to open a new PR.